pkg/policy/policy.go

```diff
@@ -49,6 +49,7 @@ const (
 	Secrets        = Target("secrets")
 	ARM            = Target("arm")
 	None           = Target("")
+	Policies       = "policies"
```

Comment:
I have avoided constants like this, I find it's better to just write the literal string "policies" rather than policy.Policies.